sympy/combinatorics/pc_groups.py

```diff
@@ -8,44 +8,71 @@ class PolycyclicGroup(DefaultPrinting):
     is_group = True
     is_solvable = True
 
-    def __init__(self, pc_sequence):
-        #self.pc_series = pc_series
+    def __init__(self, pc_sequence, pc_series):
         self.pcgs = pc_sequence
+        self.pc_series = pc_series
 
     def relative_order(self):
-        # rel_orders = []
-        # for i in range(len(self.pc_series)-1):
-        #     G = self.pc_series[i]
-        #     H = self.pc_series[i+1]
-        #     rel_orders.append(G.order()//H.order())
-        rel_orders = [g.order() for g in self.pcgs]
+        rel_orders = []
+        for i in range(len(self.pc_series)-1):
+            G = self.pc_series[i]
+            H = self.pc_series[i+1]
+            rel_orders.append(G.order() // H.order())
```

Comment:
This the order of `G/H` which is abelian but not necessarily cyclic. Relative orders of its generators are usually its proper factors.

Reply:
We are computing it from polycyclic series which has been built on cyclic factors.

Reply:
OK, assuming that the pc series is computed somewhere.